zinit.zsh

```diff
@@ -1348,7 +1348,7 @@ builtin setopt noaliases
         )
     }
     # Remove leading whitespace and trailing /.
```

Comment:
The substitution does not remove trailing `/`.

<img width="819" alt="Screenshot 2022-11-19 at 22 43 44" src="https://user-images.githubusercontent.com/10052309/202886133-ac97e7c7-1eaa-47af-a734-3f488d0d6956.png">

Reply:
I've switched to extendedglob features instead of this bash trick, to properly trim b/e whitespace and also any number of trailing `/`.